src/main/java/org/broadinstitute/hellbender/tools/walkers/bqsr/BaseRecalibrator.java

```diff
@@ -216,11 +216,8 @@ private void quantizeQualityScores() {
     }
 
     private void generateReport() {
-        try ( PrintStream recalTableStream = new PrintStream(recalTableFile) ) {
+        try ( PrintStream recalTableStream = new PrintStream(recalTableFile.getOutputStream()) ) {
             RecalUtils.outputRecalibrationReport(recalTableStream, recalArgs, quantizationInfo, recalibrationEngine.getFinalRecalibrationTables(), recalibrationEngine.getCovariates());
         }
-        catch (final IOException e) {
-            throw new UserException.CouldNotCreateOutputFile(recalTableFile, e);
```

Comment:
If we're no longer calling methods that throw `IOException`, then how are I/O errors being handled? `PrintStream` also seems to have a `checkError()` method...but in general it would be preferable to call methods that throw `IOException` instead of methods that swallow the exception and set an error flag.

Reply:
Same with this one - it should always have had the checkError call.

Reply:
I'm adding a checkError call locally to check the success of the newly added PrintStream(InputStream) constructor. The entire rest of `RecalUtils` already has this same issue (not callng checkError), so I've added an item to the [uri ticket] since to keep this PR from expanding too much.